app/model/CampaignUniquesItem.scala

```diff
@@ -0,0 +1,32 @@
+package model
+
+import ai.x.play.json.Jsonx
+import com.amazonaws.services.dynamodbv2.document.Item
+import play.api.Logger
+import play.api.libs.json.{Format, JsValue, Json}
+
+import scala.util.control.NonFatal
+
+case class CampaignUniquesItem (
+                                   campaignId: String,
+                                   startDate:Long,
+                                   endDate:Long,
+                                   reportExecutionTimestamp: String,
+                                   uniques: Long
+                                 ){
+  def toItem = Item.fromJSON(Json.toJson(this).toString())
+}
+
+object CampaignUniquesItem {
+  implicit val CampaignUniquesItemFormat: Format[CampaignUniquesItem] = Jsonx.formatCaseClass[CampaignUniquesItem]
+  def fromJson(json: JsValue) = json.as[CampaignUniquesItem]
+
+  def fromItem(item: Item) = try {
```

Comment:
Yeah I have it written down. We should also use Circe for json - much cleaner and less boilerplate.